functions/tsconfig.json

```diff
@@ -3,11 +3,12 @@
         "module": "commonjs",
         "noImplicitReturns": true,
         "noUnusedLocals": true,
-        "outDir": "lib",
+        "outDir": "./lib",
         "sourceMap": true,
         "strict": true,
         "target": "es2017"
     },
     "compileOnSave": true,
-    "include": ["src"]
+    "include": ["src", "src/**/*.ts", "src/*.ts"],
+    "exclude": ["node_modules"]
```

Comment:
are you sure the `node_modules` are being excluded? 

- if yes, this is fine
- if no, include:
`"compilerOptions": {
    "skipLibCheck": true,
    ...
 }`

Reply:
Yea, I should have added the skipLibCheck. I added it and merged it. Thanks Aidan!